Basic-Recruit-Training/Person.cpp

```diff
@@ -0,0 +1,44 @@
+#include "Person.h"
+#include <iostream>
+#include <string>
+Person::Person(){
+    age_=new int(0);
+    name_="";
+}
+Person::Person(const std::string& name, int age){
+  age_=new int(age);
+  name_=name;
+
+}
+Person::~Person(){
+    delete age_;
+}
+int Person::getAge()const{
+ return *age_;
```

Comment:
Indent your code inside functions/loops.
There's a hot debate on using tabs or spaces for indentation, and both have their merits but for the record, the team uses 4 spaces to indent code. In most text editors, there should be a setting that changes the behavior of the tab key.